pkg/extensions/extension-search-disabled.go

```diff
@@ -0,0 +1,31 @@
+//go:build !search && !ui_base
+// +build !search,!ui_base
+
+package extensions
+
+import (
+	"github.com/gorilla/mux"
+	distext "github.com/opencontainers/distribution-spec/specs-go/v1/extensions"
+	"zotregistry.io/zot/pkg/api/config"
+	"zotregistry.io/zot/pkg/log"
+	"zotregistry.io/zot/pkg/storage"
+)
+
+// EnableSearchExtension ...
+func EnableSearchExtension(config *config.Config, log log.Logger, rootDir string) {
+	log.Warn().Msg("skipping enabling search extension because given zot binary doesn't support " +
+		"this extension, please build a binary that includes this feature")
+}
+
+// SetupSearchRoutes ...
+func SetupSearchRoutes(conf *config.Config, router *mux.Router,
+	storeController storage.StoreController, log log.Logger,
+) {
+	log.Warn().Msg("skipping setting up search routes because given zot binary doesn't support " +
+		"search extension, please build a binary that includes this feature")
+}
+
+// GetExtensions...
+func GetExtensions(config *config.Config) distext.ExtensionList {
```

Comment:
Is this used?

Reply:
Used in pkg/api/routes.go, line 1269.
extensionList := ext.GetExtensions(rh.c.Config)